client/src/components/reservation/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import {
+  ReservationSection,
+  ReservationTitle,
+  ReservationInput,
+  ReservationSelect,
+  ReservationOption,
+  ReservationButton,
+  Form,
+} from "./reservation";
+import { FlexColumnContainer, FlexCenter } from "../../box/flexBox";
+import { BlockSubTitle } from "../../text/text";
+
+const Reservation = () => {
+  const [inputValue, setInputValue] = useState<string>("");
+  const [timeValue, setTimeValue] = useState<string>("2 PM");
+  const [personValue, setPersonValue] = useState<string>("2 person");
```

Comment:
It's the same issue like you did with the price, '2$' - it's not correct. Type should be number here, 'person' is just a unit for it.

Reply:
i understood about it